lib/channel-websocket/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
+  "jsnext:main": "src/index.ts",
```

Comment:
I think we need to remove this line possibly?

Reply:
This is related to tree-shaking, we need to make sure we don't blow up the bundle when removing this.

Reply:
I added this todo to the organization task